### ISSUE TITLE
Vgg16 进度更新

### ISSUE THREAD

**Issue by Aaron-Zhao123** [Aaron-Zhao123/mayo-dev#8]
Vgg16 training from scratch requires shallow -> deep training.

The current training strategy is to perturb a pretrained model.

log current status:
epoch: 4.39 | loss: 1.65    ± 21% | acc:  60.7% | ckpt: 4.25 | tp:   88/s

The goal is to obtain top 1 70% and top 5 90 %.

**Comment by Aaron-Zhao123** [comment]
epoch: 15.17 | loss: 1.51    ± 22% | acc:  64.1% | ckpt: 14.99 | tp:   75/s

**Comment by Aaron-Zhao123** [comment]
⠏ epoch: 20.91 | loss: 1.47    ± 23% | acc:  64.7% | ckpt: 20.73 | tp:   76/s

**Comment by Aaron-Zhao123** [comment]
Stopped training at:
⣿ epoch: 25.51 | loss: 1.5     ± 22% | acc:  64.2% | ckpt: 25.48 | tp:   77/s                                 

Eval report:
-       top1: 68.35%, top5: 88.52%

Now I am turning down learning rate for a top5 acc that is above 90%
